variants/classical/phase_message_test.go

```diff
@@ -79,9 +79,9 @@ func TestPhaseMessage(t *testing.T) {
 
 	s = Blank(NewPhase(1903, godip.Fall, godip.Adjustment))
 	s.SetSC("lon", godip.France)
-	s.SetSC("ber", godip.France)
-	s.SetSC("mos", godip.France)
-	s.SetSC("con", godip.France)
+	s.SetSC("par", godip.France)
+	s.SetSC("bre", godip.France)
+	s.SetSC("mar", godip.France)
```

Comment:
Ha! Nice fix :+1:

Reply:
Hah yeah. I didn't, as you may notice, have the energy to build an exhaustive test suite :/